Squest/utils/disconnect_signals.py

```diff
@@ -4,14 +4,10 @@
 def skip_auto_calculation(func):
```

Comment:
do we still need this if we have delta in "on_change" of ResourceAttribute model